test/snapp_auction.js

```diff
@@ -517,6 +517,24 @@ contract("SnappAuction", async (accounts) => {
 
       assert.equal(currentAuction.size, 2)
     })
+
+    it("Order Encoding", async () => {
```

Comment:
nit: it's always nice when this reads as "it <does something>", Here I'd suggest "encodes order information in emitted event"

Reply:
Yes, that is cool I will make the change, but this doesn't mean the `it` in it refers to "it". I always assumed it meant "item".

Reply:
```suggestion
    it("encodes order information in emitted event", async () => {
```